src/components/Reports/ReportsPageRanking/ReportsPageRanking.test.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { render, cleanup, wait } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+import DopplerIntlProvider from '../../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import ReportsPageRanking from './ReportsPageRanking';
+import { AppServicesProvider } from '../../../services/pure-di';
+
+const dateTo = new Date();
+const dateFrom = new Date();
```

Comment:
I don't undestand if this dates are being modified or now, if not, only create one, if they are being modified, maybe should you use let?

Reply:
Yes i only create two to diferenciate but i can use one